### PR TITLE
Fix "and x more" parameter list

### PR DESCRIPTION
**Pull request by nickvergessen**
@LukasReschke

### CODE REVIEW

File: tests/parameterhelpertest.php [hunk 1]
```diff
@@ -142,10 +142,45 @@ public function testPrepareParameters($params, $filePosition, $stripPath, $highl
...
+				array('A/B.txt', 'C/D.txt', 'E/F.txt', 'G/H.txt', 'I/J.txt', 'K/L.txt', 'M/N.txt'), 'file', true, false,
+				(string) $en->n('%s and %n more', '%s and %n more', 4, [implode((string) $en->t(', '), ['B.txt', 'D.txt', 'F.txt'])])
+			),
+			array(
+				array('A/B.txt', 'C/D.txt', 'E/F.txt', 'G/H.txt', 'I/J.txt', 'K/L.txt', 'M/N.txt'), 'file', true, true,
+				(string) $en->n(
+					'%s and <strong class="tooltip" title="%s">%n more</strong>',
+					'%s and <strong class="tooltip" title="%s">%n more</strong>',
+					4,
+					[
```
Comment (LukasReschke) [inline tests/parameterhelpertest.php]: `array(` for PHP 5.3 compliance of the test suite please

**Comment by nickvergessen** [comment]
yay, stable7 tests are broken, will fix in a bit.

File: tests/parameterhelpertest.php (hunk 1 above)
Comment (nickvergessen) [inline tests/parameterhelpertest.php]: Fixed

**Comment by nickvergessen** [comment]
> There was 1 error:
1) OCA\Activity\Tests\ApiTest::testGet with data set #0 ('activity-api-user2', 0, 30, array())
OC\User\NoUserException: 
/home/travis/build/owncloud/core/lib/private/files/filesystem.php:334
/home/travis/build/owncloud/core/apps/files_encryption/lib/util.php:85
/home/travis/build/owncloud/core/apps/files_encryption/lib/proxy.php:335
/home/travis/build/owncloud/core/apps/files_encryption/lib/proxy.php:316
/home/travis/build/owncloud/core/lib/private/fileproxy.php:124
/home/travis/build/owncloud/core/lib/private/files/view.php:953
/home/travis/build/owncloud/core/lib/private/files/node/node.php:158
/home/travis/build/owncloud/core/lib/private/files/node/node.php:55
/home/travis/build/owncloud/core/lib/private/files/node/folder.php:183
/home/travis/build/owncloud/core/lib/private/server.php:303
/home/travis/build/owncloud/core/lib/private/util.php:126
/home/travis/build/owncloud/core/lib/private/user/manager.php:270
/home/travis/build/owncloud/core/lib/private/user.php:184
/home/travis/build/owncloud/core/apps/activity/tests/apitest.php:33

I can not reproduce this failure travis is seeing, can you? @LukasReschke 

Also it seems to be unrelated, so wouldn't mind merging without

**Comment by LukasReschke** [comment]
Tests are passing locally.

**Comment by nickvergessen** [comment]
Merging then

**Pull request merged**